src/EnvironmentManagement/EnvironmentManager.py

```diff
@@ -326,6 +326,44 @@ def manage_removal_from_game_for(self,
             return True
         else:
             return False
+        
+    def manage_removal_from_car_for(self,
+                                     player_id: str) -> bool:
+        
+        player_was_removed = (self.__remove_and_replace(player_id))
+
+        if player_was_removed:
+            self.update_staff_ui()
+            return True
+        else:
+            return False
+
+    def __remove_and_replace(self, player_id: str) -> bool:
```

Comment:
Docstring for __remove_and_replace is missing.

Method name should be refined, e.g.: __switch_player_cars

Reply:
Resolved in f8266a8bfeaed11feb5b6230b9ebe51418b1ee90